tests/e2e/e2e.sh

```diff
@@ -75,25 +31,36 @@ ISTIO_DIR="${GOPATH}/src/istio.io/istio"
 # Create a clone of the Istio repository
 if [[ ! -d "${ISTIO_DIR}" ]]
 then
-  git clone https://github.com/sdake/istio.git "${ISTIO_DIR}"
+  git clone https://github.com/istio/istio.git "${ISTIO_DIR}"
 fi
 
 # Create an operator manifest from the default control plane configuration
-
-operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" )
+cd "${GOPATH}/src/istio.io/operator"
+operator_manifest_files=( "deploy/namespace.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_crd.yaml" "deploy/service_account.yaml" "deploy/clusterrole.yaml" "deploy/clusterrole_binding.yaml" "deploy/service.yaml" "deploy/operator.yaml" "deploy/crds/istio_v1alpha2_istiocontrolplane_cr.yaml" )
 
 # Generate the main manifest
 rm -f "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
 for manifest_file in "${operator_manifest_files[@]}"
 do
 	cat "${manifest_file}" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
-	echo "---" >> "${ISTIO_DIR}"/install/kubernetes/istio-operator.yaml
 done
 
-kubectl get pods --all-namespaces -o wide
+
+#kind cluster setup
+pushd "${ISTIO_DIR}"
+# shellcheck disable=SC1091
+source "./prow/lib.sh"
+setup_kind_cluster kindest/node:v1.15.3
+popd
+
+KUBECONFIG=$(kind get kubeconfig-path --name="istio-testing")
+export KUBECONFIG
+
+setup_docker
+
+
 
 pushd "${ISTIO_DIR}" || exit
   make istioctl
-
-  HUB=gcr.io/istio-testing TAG=latest E2E_ARGS="--use_operator --test_logs_path=${ARTIFACTS}" make e2e_simple_run
-popd
+  HUB=istio-testing TAG=1.5-dev E2E_ARGS="--use_operator --use_local_cluster=true --test.run TestSimple1 --test_logs_path=${ARTIFACTS}" make e2e_simple_noauth_run
```

Comment:
`--test.run . TestSimple1` is a subset of the test cases that should be run.  Recommend removing and fixing the deficiencies in istio/istio or this repo.

Reply:
Let's first make sure our e2e can run.  This will not make the operator build dependent on istio. This is just a test dependency as we will depend on istio for e2e test anyway.